releasenotes/notes/deprecate-complex-amp-41381bd9722bc878.yaml

```diff
@@ -0,0 +1,17 @@
+---
+deprecations:
+  - |
+    Initializing a :class:`~qiskit.pulse.library.ScalableSymbolicPulse` with complex `amp` value is now deprecated.
+    Particularly, the following pulses
+
+      * :class:`~qiskit.pulse.library.Gaussian`
+      * :class:`~qiskit.pulse.library.GaussianSquare`
+      * :class:`~qiskit.pulse.library.Drag`
+      * :class:`~qiskit.pulse.library.Constant`
+
+    initialize a :class:`~qiskit.pulse.library.ScalableSymbolicPulse`, and therefore initializing them with complex
```

Comment:
A class initializing other class is bit implementation detail and might be hard for users to understand. I think we can just say "this change also affects following pulses". But this does't block the PR and can go as is if you think it's better.

Reply:
You're right. I'll change it. Thanks.